eng/Versions.props

```diff
@@ -2,12 +2,24 @@
   <!--Package versions-->
   <PropertyGroup>
     <MicrosoftDotnetSdkInternalPackageVersion>6.0.100-preview.6.21313.2</MicrosoftDotnetSdkInternalPackageVersion>
-    <MicrosoftNETCoreAppRefPackageVersion>6.0.0-preview.6.21306.1</MicrosoftNETCoreAppRefPackageVersion>
+    <MicrosoftNETCoreAppRefPackageVersion>6.0.0-preview.6.21307.1</MicrosoftNETCoreAppRefPackageVersion>
     <MicrosoftAndroidSdkWindowsPackageVersion>30.0.100-ci.main.40</MicrosoftAndroidSdkWindowsPackageVersion>
     <MicrosoftMacCatalystSdkPackageVersion>15.0.100-ci.main.25</MicrosoftMacCatalystSdkPackageVersion>
     <MicrosoftiOSSdkPackageVersion>15.0.100-ci.main.25</MicrosoftiOSSdkPackageVersion>
     <MicrosofttvOSSdkPackageVersion>15.0.100-ci.main.25</MicrosofttvOSSdkPackageVersion>
     <MicrosoftmacOSSdkPackageVersion>12.0.100-ci.main.25</MicrosoftmacOSSdkPackageVersion>
+    <MicrosoftExtensionsHostingAbstractions>12.0.100-ci.main.25</MicrosoftExtensionsHostingAbstractions>
```

Comment:
I assume these are also just "random" values that will be overwritten by Maestro when it comes around.

Reply:
You need to suffix these with either "Version" or "PackageVersion".

Also it won't automatically fix what doesn't need updating in the version.details.xml file, so keep that in mind.

`darc verify` can tell you what might be wrong.

Reply:
Yes thanks, it's fixed and versions updated now